src/clojure_tictactoe/game/game_rules.clj

```diff
@@ -8,10 +8,10 @@
   (apply map list (horizontal-win-options board)))
 
 (defn diagonal-win-options [board]
-  (let [side-length (board/side-length board)]
-    (let [top-left-to-bottom-right-diagonal (take side-length (range 0 (count board) (+ 1 side-length)))
-          bottom-left-to-top-right-diagonal (take side-length (range (- side-length 1) (count board) (- side-length 1)))]
-      (list top-left-to-bottom-right-diagonal bottom-left-to-top-right-diagonal))))
+  (let [side-length (board/side-length board)
+        top-left-to-bottom-right-diagonal (take side-length (range 0 (count board) (+ 1 side-length)))
+        bottom-left-to-top-right-diagonal (take side-length (range (- side-length 1) (count board) (- side-length 1)))]
```

Comment:
I think these can just be called "left-diagonal" and "right-diagonal"

Reply:
Probably, but I couldn't remember which one was which when I did that, so this is more for my benefit.

Reply:
Does it matter which is which?

Reply:
I guess not.